src/Compilers/Core/CodeAnalysisTest/Text/TextChangeTests.cs

```diff
@@ -598,5 +598,23 @@ public void TestLargeTextWriterDoesNotResuseSmallChunks()
             Assert.Equal(1, GetChunks(newText).Length);
             Assert.NotSame(chunk1, GetChunks(newText)[0]);
         }
+
+        [Fact]
+        [WorkItem(10452, "https://github.com/dotnet/roslyn/issues/10452")]
+        public void TestEmptyChangeAfterChange()
+        {
+            SourceText.From("Hello World")
+                .WithChanges(new TextChange(new TextSpan(5, 6), string.Empty)) // prepare a ChangedText instance
+                .WithChanges(); // this should not cause exception
```

Comment:
@jasonmalinowski There are already many tests that check the content result of making changes. These tests were added for regression because they are typical of the actual repro.  Adding logic to assert the content would take away from the intent of the test (to show they call does not throw), by diluting it with other checks.